core-to-plc/src/Language/Plutus/CoreToPLC.hs

```diff
@@ -731,38 +937,13 @@ mkTyForall v body = do
 mkTyLam :: Converting m => GHC.Var -> m PLCType -> m PLCType
 mkTyLam v body = do
     let ghcName = GHC.getName v
-    (k', t') <- convTyVarFresh v
+    (t', k') <- convTyVarFresh v
     body' <- local (\c -> c {ccScopes=pushTyName ghcName t' (ccScopes c)}) body
     pure $ PLC.TyLam () t' k' body'
 
-
--- See Note [Recursion with Z]
-delayFunction :: MonadQuote m => PLCType -> PLCExpr -> m PLCExpr
```

Comment:
This isn't related to the abstract datatypes implementation, right? It might be clearer to move it to a separate commit (and the next two deletions below)

Reply:
Yeah, they're dead. I think I did have them in a separate commit at some point, but I must have rebased it in.